core-plugin/src/com/google/gct/idea/debugger/CloudBreakpointHandler.java

```diff
@@ -79,8 +79,9 @@ public CloudBreakpointHandler(@NotNull CloudDebugProcess process) {
    * Called when the user "reactivates" an existing snapshot to create a new breakpoint, this clones state into a new
    * breakpoint. It is different from "createIdeRepresentationsIfNecessary" in the following respects:
    * <p/>
-   * 1. it only operates on final state breakpoints. 2. it always clones them into a new IDE breakpoint 3. it does not
-   * set "created by server = true", so when control flow comes back into register, it will register with the server.
+   * 1. it only operates on final state breakpoints.
+   * 2. it always clones them into a new IDE breakpoint
```

Comment:
Should these bullet points be capitalized?  Also 2) is missing a period at the end.